connectors/src/connectors/slack/chat/citations.ts

```diff
@@ -18,11 +19,36 @@ export function annotateCitations(
   content: string,
   action: AgentActionType | null
 ): { formattedContent: string; footnotes: SlackMessageFootnotes } {
-  const references: { [key: string]: RetrievalDocumentType } = {};
+  const references: {
+    [key: string]: {
+      link: string;
+      text: string;
+    };
+  } = {};
 
   if (action && isRetrievalActionType(action) && action.documents) {
     action.documents.forEach((d) => {
-      references[d.reference] = d;
+      references[d.reference] = {
+        link: makeDustAppUrl(
```

Comment:
I know this was pre-existing code but does that mean we link to the builder from Slack? I'm pretty sure we don't in practice?